infra-library.sh

```diff
@@ -123,9 +123,14 @@ function create_gke_test_cluster() {
   local -n _custom_flags=$1
   local -n _test_command=$2
 
-  # We are disabling logs and metrics on Boskos Clusters by default as they are not used. Manually set ENABLE_GKE_TELEMETRY to true to enable logging
-  if [[ "$ENABLE_GKE_TELEMETRY" == "true" ]]; then
+  # We are disabling logs and metrics on Boskos Clusters by default as they are not used. Manually set ENABLE_GKE_TELEMETRY to true to enable telemetry
```

Comment:
The `if...else` here is a bit hard to read. I would recommend changing them to something like below:
```bash
local extra_gcloud_flags=""
if [[ "$ENABLE_GKE_TELEMETRY" == "true" ]]
  extra_gcloud_flags="${extra_gcloud_flags} --logging=NONE --monitoring=NONE"
fi
if [[ "$DISABLE_PREEMPTIBLE_NODES" == "true" ]]
  extra_gcloud_flags="${extra_gcloud_flags} --preemptible"
fi
run_kntest kubetest2 gke "${_custom_flags[@]}" --test-command="${_test_command[*]}" --extra-gcloud-flags="${extra_gcloud_flags}"
```

Reply:
Fixed, @dprotaso I made the preemptible opt-in